src/modules/design/bpmn-io/bpmn-io.ts

```diff
@@ -10,15 +10,19 @@ import {
   IDiagramExportService,
   IDiagramPrintService,
   IEditorActions,
+  IElementRegistry,
+  IEvent,
   IEventFunction,
   IKeyboard,
+  IShape,
   NotificationType,
 } from '../../../contracts/index';
 import environment from '../../../environment';
 import {NotificationService} from '../../notification/notification.service';
 import {DiagramExportService, DiagramPrintService} from './services/index';
 
 const sideBarRightSize: number = 35;
+const elementRegistryTimeoutMilliSeconds: number = 50;
```

Comment:
Milliseconds is one word
```suggestion
const elementRegistryTimeoutMilliseconds: number = 50;
```